### DISCUSSION TITLE
[Feature] Remove and re tag face for wrongly tagged face in the mobile app

### DISCUSSION THREAD

**Discussion by k1RAN02** [immich-app/immich#14858] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The ability to do more in the mobile app instead of logging into the URL, like retag people who have been misidentified.
The ability to run some jobs, like resetting the face database on the app without turning on the link.
Choosing a better feature photo than blurred from automatically like photos app in macOS or ios. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is a bunch of feature requests, and some duplicates.
